app/src/main/java/com/mainApp/yelp_mini/model/repo/RepoClass.kt

```diff
@@ -0,0 +1,38 @@
+package com.mainApp.yelp_mini.model.repo
+
+import androidx.lifecycle.MutableLiveData
+import com.mainApp.yelp_mini.model.data.YelpRestaurantDetail
+import com.mainApp.yelp_mini.model.data.YelpRestaurantReviews
+import com.mainApp.yelp_mini.model.data.YelpSearchResult
+
+class RepoClass {
+    private val apiCall = APIClass()
+    private var restaurants: MutableLiveData<YelpSearchResult?> = MutableLiveData()
+    private var restaurantsDetailList: MutableLiveData<YelpRestaurantDetail?> = MutableLiveData()
+    private var restaurantsReviewList: MutableLiveData<YelpRestaurantReviews?> = MutableLiveData()
+
+    fun getRestaurantReview(restaurantID: String): MutableLiveData<YelpRestaurantReviews?> {
```

Comment:
Use `LiveData` instead of `MutableLiveData` as return type. Also implement this in all places in this class